mesheryctl/internal/cli/root/components/view.go

```diff
@@ -58,6 +58,8 @@ mesheryctl component view [component-name]
 			log.Fatalln(err, "error processing config")
 		}
 
+		outFormatFlag, _ = cmd.Flags().GetString("output-format")
```

Comment:
```suggestion
		outFormatFlag, _ := cmd.Flags().GetString("output-format")
```

also can yuou remove the variable declaration in component.go file after it will be ok to merge